README.md

```diff
@@ -1 +1,26 @@
-Terraform configuration for kafka clusters to create topis and ACLs
+# Terraform configuration for kafka clusters to create topis and ACLs
+
+## how to plan/ apply shared Kafka Terraform resources?
+
+1. Login to AWS- run `make okta-aws-cli-login` for profile `pubsub-admin` from the `terraform` repo dir. More
+   info [here](https://github.com/utilitywarehouse/terraform/tree/master/aws#okta-login)
+
+```
+[pubsub]
+region         = eu-west-1
+role_arn       = arn:aws:iam::950135041896:role/pubsub-admin
```

Comment:
Should this role be called `pubsub-dev` and you can define `pubsub-prod` as well that points to the uw-prod `pubsub-admin` role?